src/Plaid/SerializableContent.cs

```diff
@@ -20,7 +20,7 @@ public virtual string ToJson()
 		{
 			return JsonConvert.SerializeObject(this, new JsonSerializerSettings()
 			{
-				DateFormatString = "yyyy-MM-dd",
+				DateFormatString = "yyyy-MM-ddTHH:mm:ssZ",
```

Comment:
Why is this change necessary? Could this have unintended side effects for other parts of the code?

Reply:
That's true, it could have unintended side effects. https://github.com/LN-Zap/Plaid.NET/pull/6/commits/e391485cc83fbf4c4074b4f5aa66a9af65e018ce
It's necessary because the docs say they expect an ISO-8601 date BTW https://plaid.com/docs/api/products/#accounts-balance-get-request-min-last-updated-datetime_options